rocksdb_admin/admin_handler.h

```diff
@@ -134,7 +134,8 @@ class AdminHandler : virtual public AdminSvIf {
   bool clearMetaData(const std::string& db_name);
   bool writeMetaData(const std::string& db_name,
                      const std::string& s3_bucket,
-                     const std::string& s3_path);
+                     const std::string& s3_path,
+                     const int64_t& last_kafka_msg_timestamp_ms);
```

Comment:
remove &, copying an int64_t is equally expensive to copying a reference, while accessing through reference introduces an unnecessary indirection.

set a default value for the new parameter.

Reply:
Done, thanks for the explanation.